ulid2.py

```diff
@@ -189,7 +192,13 @@ def generate_binary_ulid(timestamp=None):
     ts_bytes = _to_binary(
         (ts >> shift) & 0xFF for shift in (40, 32, 24, 16, 8, 0)
     )
-    return ts_bytes + os.urandom(10)
+    entropy = os.urandom(10)
```

Comment:
Could the generation of Truly Monotonic Entropy (not a registered trademark) be refactored into another function, so it's less tangled with the stuff here?

Reply:
What about adding `ensure_monotonic` as a kwarg to the relevant `generate_*_ulid` functions and 

```python
entropy = os.urandom(10)
if ensure_monotonic and _last_timestamp == ts and _last_entropy is not None:
     while entropy < _last_entropy:
         entropy = os.urandom(10)
```

Reply:
check out my edits - it's at least behind a kwarg now.